lambda/lambda.js

```diff
@@ -99,6 +99,11 @@ async function handler(request) {
                     resolve(false);
                     return;
                 }
+                if (Math.floor(Date.now() / 1000) + 3600 < payload.exp) {
+                    console.log(jwtKey.modeName + ': \'Expiration time\' (exp) claim is too far in the future ');
+                    resolve(false);
+                    return;
+                }
                 console.log(jwtKey.modeName + ": payload " + JSON.stringify(payload, null, '  '));
```

Comment:
It seems that this line should be called before the expiration check to make sure we log the payload independently from the check results (the same we do if the verification fails)